eng/targets/CSharp.Common.targets

```diff
@@ -14,21 +14,21 @@
     <!-- Public members should not use oblivious types. Not done with all nullable annotations. -->
     <NoWarn>$(NoWarn);RS0041</NoWarn>
   </PropertyGroup>
-  <ItemGroup Condition=" '$(IsImplementationProject)' == 'true' AND
+
+  <ItemGroup Condition=" Exists('$(MSBuildProjectDirectory)\PublicAPI.*.txt') AND
+      '$(IsImplementationProject)' == 'true' AND
```

Comment:
New version emits errors if Public API baseline files are missing

Reply:
Nice - would be good to revert this once we can fix all the occurrences.

Reply:
Agree. If we undid this change and eliminated analyzers and `msbuild` tasks, we'd know for sure the public API is covered. Might need to invent a special `ImNotPubliclyConsumable` flag for Microsoft.Web.Xdt.Extensions and other special cases😺

First, we need to be able to analyze projects containing `*.cshtml` or `*.razor` files:
- Microsoft.AspNetCore.Authentication.AzureAD.UI
- Microsoft.AspNetCore.Authentication.AzureADB2C.UI
- Microsoft.AspNetCore.Components.Authorization
- Microsoft.AspNetCore.Identity.UI

Do we have an issue tracking this problem❔ /cc @sharwell @mavasani

Reply:
https://github.com/dotnet/roslyn/issues/47350 is tracking it, but it might need to be fixed in aspnetcore.